src/Okta.Sdk/Generated/IPasswordPolicyConditions.Generated.cs

```diff
@@ -13,7 +13,7 @@
 namespace Okta.Sdk
 {
     /// <summary>Represents a PasswordPolicyConditions resource in the Okta API.</summary>
-    public partial interface IPasswordPolicyConditions : IResource
+    public partial interface IPasswordPolicyConditions : IPolicyRuleConditions
```

Comment:
TODO: Double check. Should policyConditions class inherits from PolicyRuleConditions?